### ISSUE TITLE
Details for training

### ISSUE THREAD

**Issue by riolys** [TreeLLi/APT#1]
Could you plz share more training details in the paper.
For example, in Figure 5, some details are not shown, 
(1)the seed chosen for each dataset, this is important since different seed varies in performance;
(2)use which of the "constant", "on-the-fly" or "perturbed". If the last is used, how should I choose  _step size alpha_;
I will appreciate if you supply more details. More checkpoint files and settings is better(the given ones only include shot=1 and seed=3 setting)!

**Comment by TreeLLi** [comment]
Hi, thanks for your interest in our work! For your requests, 

1. We did not use the same seeds for different datasets and training set-ups so it's inconvenient to provide the specifc seed given the large amount of experiments we run. Do you have a particular set of experiments you want to have seeds of?
2. We used "on-the-fly" by default. If you are interested in using "perturbed" strategy, a sensitivity test on alpha' is given in Table 9.
3. We are happy to provide more checkpoints. Do you have specific ones you want? By the way, the current released checkpoints are for shot=-1(All) instead of 1.
4. What else do you need for the training setting?

**Comment by riolys** [comment]
Thanks! I'd like to reproduce your code(Figure 5) since APT is very interesting based on CoOp.
In the paper, the settings are explicit. I wonder if the seeds are different for each dataset and each shot number M.
Could you share 1,4,16 shots of APT checkpoints and logs(or settings) in Figure 5?

**Comment by TreeLLi** [comment]
Hi, yes, the seeds are slightly different. The checkpoints alongside the logs and settings of all involved data shots for one run are available now through the original link.
